inference-engine/src/inference_engine/CMakeLists.txt

```diff
@@ -168,8 +168,8 @@ add_library(${TARGET_NAME} SHARED
 set_ie_threading_interface_for(${TARGET_NAME})
 
 target_link_libraries(${TARGET_NAME} PRIVATE ${CMAKE_DL_LIBS} Threads::Threads pugixml openvino::itt
-                                             ${NGRAPH_LIBRARIES} inference_engine_transformations
-                                     PUBLIC ${TARGET_NAME}_legacy)
+                                             inference_engine_transformations
+                                     PUBLIC ${TARGET_NAME}_legacy ${NGRAPH_LIBRARIES})
```

Comment:
later in this file we need to export `NGRAPH_LIBRARIES` to `IE` namespace:
```cmake
export(TARGETS ngraph ${TARGET_NAME} NAMESPACE IE:: APPEND FILE "${CMAKE_BINARY_DIR}/targets.cmake")
```
This should allow Jenkins to pass because demo build step uses local IE config file (`config.cmake.in`):
```cmake
-DInferenceEngine_DIR=C:\j\workspace\private-ci\ie\build-windows-vs2019\b\build\dldt 
```
 which should be fixed by that line.